runtime/main.h

```diff
@@ -12,9 +12,15 @@
 #include "entry-names.h"
 
 namespace Fortran::runtime {
-extern int argc;
-extern const char **argv;
-extern const char **envp;
+struct ExecutionEnvironment {
```

Comment:
I noticed that this file has the wrong name in the license at the beginning of the file.